tests/sentry/api/endpoints/test_project_agnostic_rule_conditions.py

```diff
@@ -11,4 +11,4 @@ def test_simple(self):
         response = self.client.get(url, format="json")
 
         assert response.status_code == 200, response.content
-        assert len(response.data) == 9
+        assert len(response.data) == 10
```

Comment:
Should this be returning 10 items if the new feature flag is working?

Reply:
Oh, missed this! Good catch. I only modified `project_rules_configuration.py`